features/automation.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+    name: open-cluster-management-automation-template
```

Comment:
I think we can just drop `open-cluster-management` here because the label kind of already indicates that.  Maybe we can name this `ansible-automation-template`